docs/codeql/ql-language-reference/modules.rst

```diff
@@ -273,4 +273,69 @@ The ``<module_expression>`` itself can be a module name, a selection, or a quali
 reference. For more information, see ":ref:`name-resolution`."
 
 For information about how import statements are looked up, see "`Module resolution <https://codeql.github.com/docs/ql-language-reference/ql-language-specification/#module-resolution>`__"
-in the QL language specification. 
\ No newline at end of file
+in the QL language specification. 
+
+Built-in modules
+****************
+
+QL defines a `QlBuiltins` module, which is a built-in module that is always in scope.
+Currently, it defines a single parameterized sub-module
+`EquivalenceRelation`, that provides an efficient abstraction for working with
+(partial) equivalence relations in QL.
+
+Equivalence relations
+=====================
+
+The built-in `EquivalenceRelation` module is parameterized by a type `T` and a
+binary base relation `eq` on `T`. The symmetric and transitive closure of `eq`
```

Comment:
I don't think `eq` is a good name for the relation that's _not_ an equivalence relation. Other possible names: `base`, `baseRelation`, `rel`. I like `base`, and that's also what you're using in the example below.